app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -142,6 +134,10 @@
       }
     }
 
+    .hide-button {
```

Comment:
(nitpick, non-blocking): I feel like adding a shared utility class like `invisible` to compliment our `hidden` utility class may be useful.

Reply:
Yup definitely, I marked it as non blocking so that it doesn't hold back the PR. I can look it into tomorrow to see why it didn't work.